package.json

```diff
@@ -29,5 +30,9 @@
     "ethjs-ens": "^2.0.1",
     "ipfs-http-client": "^39.0.2",
     "semver": "^5.5.0"
-  }
+  },
+  "devDependencies": {
+    "parcel-bundler": "^1.12.4"
```

Comment:
Parcel is not really meant for npm packages, but more suited for bundled applications.

We may want to use [microbundle](https://github.com/developit/microbundle) instead.